### ISSUE TITLE
Splitting before cross-validation

### ISSUE THREAD

**Issue by Mikeymice** [UBC-MDS/Capital_Bikeshare_Analysis#24]
It just came to me that i might used the cross validation wrong. 

apparently we are supposed to split into training and testing before using cross validation on the training to find the height. and use the height to train the training set and use the test set to compute accuracy 
https://towardsdatascience.com/train-test-split-and-cross-validation-in-python-80b61beca4b6

I will changed the script and maybe we will also output a csv file for best tree height and the accuracy on the test set

**Comment by EvanYathon** [comment]
Sounds good!  Thanks for the update.
